Migrations.cs

```diff
@@ -360,5 +360,70 @@ public int UpdateFrom8()
 
             return 9;
         }
+
+        public int UpdateFrom9()
+        {
+            _contentDefinitionManager.AlterPartDefinition(nameof(LinkBehaviourPart), builder => builder
+                .Attachable()
+                .WithDisplayName("Link Behaviour")
+                .WithDescription("Define settings for link behaviour")

```

Comment:
Change to "Define behaviour when user interacts with link."

Reply:
We also need to add `WithPosition("15")` underneath `WithDescription`. We'll also add `WithPosition("10")` to the link destination part. This is to ensure when they are attached to a content type the link destination appears above the behaviour.